gateway/src/main/java/org/georchestra/gateway/accounts/admin/CreateAccountUserCustomizer.java

```diff
@@ -81,10 +81,14 @@ public class CreateAccountUserCustomizer implements GeorchestraUserCustomizerExt
             } else {
                 user = accounts.getOrCreate(mappedUser);
             }
+            if (isPreAuth) {
+                user.setIsPreAuth(mappedUser.getIsPreAuth());
+                user.setPreauthIdp(mappedUser.getPreauthIdp());
+                user.setPreauthIdpUserId(mappedUser.getPreauthIdpUserId());
+            }
```

Comment:
@emmdurin , @pmauduit , @fvanderbiest,
Here we set user properties related to preAuth mode, without saving it into LDAP.

Reply:
just `setIsPreAuth` is needed, and maybe we should name it setIsExternalAuth to include preauthentication + OIDC.